embedding/csharp/Tizen.Flutter.Embedding/Tizen.Flutter.Embedding.csproj

```diff
@@ -1,12 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿  <Project Sdk="Tizen.NET.Sdk/1.1.7">
```

Comment:
I need opinion on whether it's okay to modify csproj like this.

Reply:
I think you should target [multiple TFMs](https://docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file) in this case, but let's discuss in more detail with @WonyoungChoi tomorrow (offline).

Example: https://stackoverflow.com/questions/42747977/how-do-you-multi-target-a-net-core-class-library-with-csproj

Reply:
It seems that just modifying the `PackageReference` version is good. Thank you for your reply.
```csproj
<PackageReference Include="Tizen.NET" Version="9.0.0.16760" />
```

Reply:
Umm, I was only saw about the failed build of this PR. For the multiple TFMs you mentioned, I'll look into it too.

Reply:
I've been testing several combinations and https://github.com/swift-kim/flutter-tizen/blob/multiple-tfms/embedding/csharp/Tizen.Flutter.Embedding/Tizen.Flutter.Embedding.csproj seems to be the best solution for now. I'll discuss further with @WonyoungChoi though.